### PR TITLE
Fix yamllint on update-deployment task

### PR DESCRIPTION
**Pull request by lcarva**
# Before you complete this pull request ...

Look for any open pull requests in the repository with the title "e2e-tests update" and 
see if there are recent e2e-tests updates that will be applicable to your change.

### CODE REVIEW

File: task/update-deployment/0.1/update-deployment.yaml [hunk 1]
```diff
@@ -41,27 +46,27 @@ spec:
...
 
       git config --global user.email "rhtap@noreplay.com"
       git config --global user.name "gitops-update"
 
-      git clone $(params.gitops-repo-url)
+      git clone ${PARAM_GITOPS_REPO_URL}
       cd ${gitops_repo_name}
 
-      sed -i 's| image: .*| image: $(params.image)|' $deployment_patch_filepath
+      sed -i "s| image: .*| image: ${PARAM_IMAGE}|"" $deployment_patch_filepath
```
Comment (MartinBasti) [inline task/update-deployment/0.1/update-deployment.yaml:67]: Isn't it and extra `"`?
```suggestion
      sed -i "s| image: .*| image: ${PARAM_IMAGE}|" $deployment_patch_filepath
```

Comment (lcarva) [reply task/update-deployment/0.1/update-deployment.yaml]: Good eye. Thank you!

**Review by MartinBasti — APPROVED** [review]
I'm not sure if I can approve this, but in case that I can, then LGTM

**Comment by lcarva** [comment]
cc @rcerven

**Pull request merged**